fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/DefaultRequest.kt

```diff
@@ -198,8 +199,8 @@ data class DefaultRequest(
      */
     override fun body(openStream: BodySource, calculateLength: BodyLength?, charset: Charset, repeatable: Boolean): Request {
         _body = DefaultBody
-            .from(openStream = openStream, calculateLength = calculateLength, charset = charset)
-            .let { body -> if (repeatable) body.asRepeatable() else body }
+                .from(openStream = openStream, calculateLength = calculateLength, charset = charset)
```

Comment:
You probably didn't want to change this indentation to 8 instead of 4.

Reply:
I think we need to sync on this! Can you share your setting and push to the project?

Reply:
Yes, but I think I just followed the klint instructions. I think the contributors file also has a few lines on how to get editorconfig working.